rasa_core_sdk/forms.py

```diff
@@ -339,9 +340,16 @@ def validate(self, dispatcher, tracker, domain):
                     "".format(slot_to_fill, self.name()),
                 )
 
-        for slot, value in slot_values.items():
+        for slot, value in list(slot_values.items()):
             validate_func = getattr(self, "validate_{}".format(slot), lambda *x: value)
```

Comment:
I guess this should be:
```suggestion
            validate_func = getattr(self, "validate_{}".format(slot), lambda *x: {slot: value})
```

Reply:
oh yes, we totally changed this before and i missed it when i merged master, thanks for the catch!

Reply:
now i'm thinking `{'slot_name': 'value'}` in my warning should actually be ` {'slot_name': value}`, since values don't have to be strings, right?

Reply:
I'd say {'{slot_name}': value}`?